lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -263,6 +263,9 @@ keys_lookup[113] = "mouse_wheel_down"
 
 registerCallback("destruct",function(self)
 	KeyAlert[self.entity] = nil
+    --Used futher below. Didn't want to create more then one of these per file
+    if spawnAlert[self.entity] then spawnAlert[self.entity] = nil; end
+    if leaveAlert[self.entity] then leaveAlert[self.entity] = nil; end
```

Comment:
also, semicolons are mostly useless in lua
also, random indentation
also, missing space after comma in line 264